### PR TITLE
Size cleanup

### LINKED ISSUE #369: chemiscope local web app size 0
I have an input file I'm creating with `write_input`, and I want to use the species of the atoms as a property to control some aspect of visualization, probably size. I add it to the `Atoms` object with `atoms.new_array("Z", atoms.numbers)`, and then interactive set size as coming from `"Z"`.  However, for Cu Al, Zs 29 and 13, I don't get circles that are off by a factor of ~2, but rather tiny specks, and reasonable size circles.

Is setting `size` from a property reasonably tested, such that this is surprising? Or is it just something that's not well supported?  Either way I can try to do a more complete bug report.

### PR DESCRIPTION
**Pull request by ceriottm**
Fixes #369. I cleaned up a bit the point-size-scaling machinery in a way that hopefully addresses that problem, and makes it more robust and self-explanatory.

### CODE REVIEW

File: src/map/options.html.in [hunk 1]
```diff
@@ -169,17 +169,15 @@
...
-                     <select id="map-size-transform" class="form-select" disabled>
-                        <option value="linear">linear</option>
-                        <option value="log">log</option>
-                        <option value="sqrt">sqrt</option>
-                        <option value="inverse">inverse</option>
+                     <select id="map-size-transform" class="form-select">
+                        <option value="linear" title="Linear mapping over the full range of values (large value -> large size)">linear</option>
+                        <option value="flip-linear" title="Linear reverse mapping over the full range of values (large value -> small size)">flipped linear</option>
+                        <option value="proportional" title="Proportional to actual value, breaks for negative values">proportional</option>
+                        <option value="log" title="Log-scale of actual value, breaks for negative values">log</option>
```
Comment (Luthaf) [inline src/map/options.html.in:176]: The title does not show for me anywhere. Can you see it in the rendered HTML?

**Review by Luthaf — COMMENTED** [review]
I'm not sure I like this. When there are negative values in the dataset, this will look like there are missing points. 

Why/where is the previous code not working?

File: src/map/options.html.in (hunk 1 above)
Comment (ceriottm) [reply src/map/options.html.in:176]: the title should (and does, for me) show up as a tooltip if you hover the mouse over the option.

**Comment by ceriottm** [comment]
> I'm not sure I like this. When there are negative values in the dataset, this will look like there are missing points.
> 
> Why/where is the previous code not working?

For instance, the log would have negative values, that would render as the absolute value so small values give larger points. Also in general, given we don't have a "scale" to refer to, it is really hard (should I say impossible?) to interpret a sqrt(x), log(x) or 1/x scaling if it is referred to a shifted range.

**Comment by Luthaf** [comment]
I guess that's fair, but then I would do like when using a log scale for the x/y/z axis and (a) send a warning to the user instead of doing this silently, and (b) ideally drop/hide the corresponding points

**Comment by ceriottm** [comment]
> I guess that's fair, but then I would do like when using a log scale for the x/y/z axis and (a) send a warning to the user instead of doing this silently, and (b) ideally drop/hide the corresponding points

Fair. Done ^_^

File: src/map/options.html.in (hunk 1 above)
Comment (Luthaf) [reply src/map/options.html.in:176]: Ah, right! It just take a second for it to appear

**Review by Luthaf — APPROVED** [review]

**Pull request merged**